### PR TITLE
style(app): Address device landing design feedback

### LINKED ISSUE #10611: 6.0 Feedback: [Design QA] T1 Device Landing Page
### Overview

_No response_

### Current Behavior

Link to [annotated frame in Figma ](https://www.figma.com/file/bVtCogdwiXj3ndpk4AWXCx/Milestone-1%3A-Read-and-Execute?node-id=13793%3A274729)

- [x] (1) Because we don't have a button in the top right of the `DevicesLanding_title` the height of the container is a bit less tall. Needs to match the other pages at a height of 36px. 
- [x] (2) Missing states on this _styling/link/secondary/grey/internal_
- [x] (3) Missing states on _btn-icon/accordion_
- [x] (4) Missing states on the _card/medium/robot_
- [x] (5) Needs to be set widths for these columns so if a pipette is empty, the columns still align across all cards 
- [x] (6) Needs a bit more spacing on these icons. I have 0.25rem between them 
- [x] (7) Needs to be `darkGreyEnabled` when enabled, and then `darkBlackEnabled` when on hover + tooltip
- [x] (8) Needs .25rem margin between header and copy across all these columns
- [x] (9) There's a margin of 8px above this button, needs to be 4px on top and 4px to the right 
- [x] (10) When I click on the button the focus state appears, but that should only appear when the user is physically tabbing through the links. The active state is displayed to the left (viewable in the figma frame) and does not include the orange outline. 
- [x] (11) Placement of the dropdown is off, needs to be directly below the button with .25rem margin between. 
- [x] (12) Should be our stroke version of the tertiary button, also remove the box-shadow! 
- [x] (13) _RobotStatusBanner_opentrons-dev_goToRun_ should be at the top of the container, currently centered within

Screen Shot 2022-06-03 at 11 01 25 AM

Screen Shot 2022-06-03 at 9 37 40 AM

Screen Shot 2022-06-03 at 9 37 19 AM

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Operating system

_No response_

### Robot setup or anything else?

_No response_

### PR DESCRIPTION
**Pull request by koji**
# Overview
This PR will fix #10611 and #10832

https://user-images.githubusercontent.com/474225/174353810-673af2bf-3a06-4811-8386-28afa88044ff.mov

# Changelog
- Update styling to align with design feedback
- Add SecondaryTertiaryButton to atoms/buttons
- Make Available & Unavailable (flex) clickable and add `css cursor: pointer`

# Review requests
- mount and modules
[before]
device_landing_before

[after]
device_landing_after

- `Go to run` button is updated on device landing and device details
https://www.figma.com/file/l7BAJAGICr10ELsSqQD0Sr/Design-System?node-id=1782%3A83236

Screen Shot 2022-06-16 at 4 06 51 PM

Screen Shot 2022-06-16 at 11 40 51 AM

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/Devices/RobotCard.tsx [hunk 1]
```diff
@@ -122,12 +130,17 @@ function AttachedModules(props: { robotName: string }): JSX.Element | null {
...
+      flexDirection={DIRECTION_COLUMN}
+      paddingRight={SPACING.spacing4}
+      width="100%"
+    >
       <StyledText
         as="h6"
         textTransform={TEXT_TRANSFORM_UPPERCASE}
         color={COLORS.darkGreyEnabled}
-        marginBottom={SPACING.spacing1}
+        // marginBottom={SPACING.spacing1}
```
Comment (vabruzzo) [inline app/src/organisms/Devices/RobotCard.tsx]: probably meant to remove?

File: app/src/pages/Devices/DevicesLanding/NewRobotSetupHelp.tsx [hunk 2]
```diff
@@ -16,6 +18,18 @@ import { Modal } from '../../../atoms/Modal'
...
+  color: ${COLORS.darkBlack};
+  opacity: 0.7;
+  font-size: ${TYPOGRAPHY.fontSizeLabel};
+  font-weight: ${TYPOGRAPHY.fontWeightSemiBold};
+  &:hover {
+    opacity: 1;
+  }
+`
+
+// const ACCORDION_STYLE = css``
```
Comment (vabruzzo) [inline app/src/pages/Devices/DevicesLanding/NewRobotSetupHelp.tsx]: remove also?

**Review by vabruzzo — APPROVED** [review]
just a couple of comments i think are unnecessary, otherwise looks good!

File: app/src/organisms/Devices/RobotCard.tsx (hunk 1 above)
Comment (koji) [reply app/src/organisms/Devices/RobotCard.tsx]: good catch!

**Comment by mmencarelli** [comment]
**Devices Landing Page Feedback**

[Link to annotated frame on Figma](https://www.figma.com/file/bVtCogdwiXj3ndpk4AWXCx/Milestone-1%3A-Read-and-Execute?node-id=14570%3A372231)

- [x] (1) Up this margin to 1.5rem 
- [x] (2) Update the button overflow icon. Work with @jerader if needed for assistance, they just unlocked it in their most recent work. 
- [x] (3) Update colour of this text to match designs
- [x] (4) When I adjust the copy to reflect our longest named pipette, it breaks onto two lines. Should be on one still!
- [ ] (5) Can these columns not go onto 2 lines unless the screen size is very small? There's space on the card still for it to be on one line
- [ ] (6) When I click 'run a protocol', it opens up the slideout but doesn't close the menu 
- [x] (7) There's a bit of extra space here. Hard for me to figure out what's going on but feel free to compare to the designs below in the figma frame
- [x] (8) Pop this fixed bottom of frame with a margin-bottom of 40px 
- [x] (9) Remove box-shadow from primary btn
- [x] (10) Nix these two (available/unavailable) when in discovery mode 
- [x] (11) Update styling and location of this link 
- [x] (12) Update size and colour of the spinner
- [x] (13) remove the '...' 

Screen Shot 2022-06-22 at 11 09 55 AM
2054/175107132-511886fa-c6cc-40d9-9930-8b7f2f344754.png">

Screen Shot 2022-06-22 at 11 10 10 AM

Screen Shot 2022-06-22 at 11 10 41 AM

Screen Shot 2022-06-22 at 11 10 46 AM
Screen Shot 2022-06-22 at 11 11 16 AM
Screen Shot 2022-06-22 at 11 11 20 AM

**Review by mmencarelli — CHANGES_REQUESTED** [review]
changes requested above!

**Comment by koji** [comment]
Screen Shot 2022-06-23 at 10 35 11 AM

**Comment by koji** [comment]
(6) needs some modification and Ayana is working on that.

**Comment by koji** [comment]
The rest, no.5 will be solved in another PR.

**Pull request merged**